frontend/js/src/common/listens/MBIDMappingModal.tsx

```diff
@@ -56,6 +58,11 @@ export default NiceModal.create(({ listenToMap }: MBIDMappingModalProps) => {
     setTimeout(modal.remove, 200);
   }, [modal]);
 
+  const massLinkButton = React.useCallback(() => {
```

Comment:
This could be renamed to describe the action, something along the lines of 
```suggestion
  const closeAndGoToMassLink = React.useCallback(() => {
```